packages/client/src/components/ExportButton.tsx

```diff
@@ -26,6 +35,13 @@ const ExportButton = () => {
 			<InfoMessage message={'not logged in'} status={RequestStatus.Failed} />
 		);
 	}
+
+	const transcriptFormatDescription: Record<TranscriptFormat, string> = {
+		srt: 'SRT format (Time Coded)',
+		text: 'TEXT format',
+		json: 'JSON format',
```

Comment:
I would remove the word 'format' from these since there is a label on the dropdown. I think TEXT should be title case 'Text' as I imagine many journalists won't be aware of TXT/ 'text' format - 'Text' seems clearer